pkg/templates/templates.go

```diff
@@ -413,6 +417,12 @@ func (t *TemplateResolver) ResolveTemplate(tmplRaw []byte, context interface{})
 		templateStr = t.processForDataTypes(templateStr)
 	}
 
+	// process for copied map data
+	if strings.Contains(templateStr, "copyConfigMapData") ||
```

Comment:
This seems to be making an existing problem worse by calling `strings.Contains` up 5 times. Could you just always call `processForDataTypes` and have `processForDataTypes` return early if the regex doesn't find any matches? Then it's just one time reading the template string rather than up to 5 times.

Reply:
This is fixed, thanks for the idea.